src/workspacer.Bar/Widgets/BatteryWidget.cs

```diff
@@ -11,8 +11,12 @@ namespace workspacer.Bar.Widgets
     public class BatteryWidget : BarWidgetBase
     {
         public Color LowChargeColor { get; set; } = Color.Red;
+        public Color MedChargeColor { get; set; } = Color.Yellow;
+        public Color HighChargeColor { get; set; } = Color.Green;
         public bool HasBatteryWarning { get; set; } = true;
         public double LowChargeThreshold { get; set; } = 0.10;
+        public double MedChargeThreshold { get; set; } = 0.50;
+        public double HighChargeThreshold { get; set; } = 1.00;
```

Comment:
This value isn't used for anything and should be removed. 

Apart from that, this all looks good to me 👍

Reply:
Changed it to only have a Low Battery Color